tests/integration/suite/daprd/jobs/grpc/basic.go

```diff
@@ -118,4 +118,50 @@ func (b *basic) Run(t *testing.T, ctx context.Context) {
 			require.NoError(t, err)
 		}
 	})
+
+	t.Run("overwrite if exists", func(t *testing.T) {
+		for _, req := range []*rtv1.ScheduleJobRequest{
+			{Job: &rtv1.Job{
+				Name:      "overwrite1",
+				Schedule:  ptr.Of("@daily"),
+				Repeats:   ptr.Of(uint32(1)),
+				Overwrite: true,
+			}}, {Job: &rtv1.Job{
+				Name:      "overwrite1",
+				Schedule:  ptr.Of("@daily"),
+				Repeats:   ptr.Of(uint32(1)),
+				Overwrite: true,
+			}},
+		} {
+			_, err := client.ScheduleJobAlpha1(ctx, req)
+			require.NoError(t, err)
+		}
+	})
+
+	t.Run("do not overwrite if exists", func(t *testing.T) {
+		r := &rtv1.ScheduleJobRequest{Job: &rtv1.Job{
+			Name:     "overwrite2",
+			Schedule: ptr.Of("@daily"),
+			Repeats:  ptr.Of(uint32(1)),
+		}}
+		_, err := client.ScheduleJobAlpha1(ctx, r)
+		require.NoError(t, err)
+
+		for _, req := range []*rtv1.ScheduleJobRequest{
+			{Job: &rtv1.Job{
+				Name:      "overwrite1",
+				Schedule:  ptr.Of("@daily"),
+				Repeats:   ptr.Of(uint32(1)),
+				Overwrite: true,
+			}}, {Job: &rtv1.Job{
+				Name:      "overwrite1",
+				Schedule:  ptr.Of("@daily"),
+				Repeats:   ptr.Of(uint32(1)),
+				Overwrite: true,
+			}},
+		} {
+			_, err := client.ScheduleJobAlpha1(ctx, req)
+			require.NoError(t, err)
+		}
+	})
```

Comment:
Please can we create a new `tests/integration/suite/daprd/jobs/grpc/overwrite.go` int test for this specific scenario.